dash/core_api.py

```diff
@@ -125,15 +110,27 @@ def move_section(id_num, space):
     response = requests.get(url)
     data = json.loads(response.text)
 
-    template["space"] = space
-    template["x"] = data["x"]
-    template["y"] = data["y"]
-    template["w"] = data["w"]
-    template["h"] = data["h"]
-    template["app"] = data["app"]
+    new_data = {
+        "space": "SpaceOne",
+        "x": 0,
+        "y": 0,
+        "w": 1440,
+        "h": 808,
+        "app": {
+            "url": APP_URL,
+            "states": {"load": {"url": f"{PLOT_URL}/plot1"}},
+        },
+    }
```

Comment:
You shouldn't need all of this because it is all overwritten immediately below. Just this will do:
```suggestion
    new_data = {}
```

Or even define the dictionary the way you have and just set the values to what is in `data` immediately.